internal/batches/docker/context.go

```diff
@@ -67,9 +67,11 @@ func (e *fastCommandTimeoutError) Error() string {
 
 func (*fastCommandTimeoutError) Timeout() bool { return true }
 
+type fastCommandTimeoutKey string
```

Comment:
Why do we need this type?

Reply:
golangci-lint reminded me that using a custom type the string is ensured to always be globally unique, if it's a plain string it might have clashes with other ctx keys - pretty interesting magic honestly

Reply:
Oh interesting.